### PR TITLE
Code cleanup

### PR DESCRIPTION
**Pull request by advaypal**

### CODE REVIEW

File: MIDIchlorians/MIDIchlorians/AppDelegate.swift [hunk 1]
```diff
@@ -148,10 +149,11 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
         appSamples.forEach { sample in
             // if saving fails, what are we gonna do?
```
Comment (anand-sundaram) [inline MIDIchlorians/MIDIchlorians/AppDelegate.swift]: Should probably remove this comment if we are not doing anything to handle failure :)

File: MIDIchlorians/MIDIchlorians/DataManager.swift [hunk 2]
```diff
@@ -16,14 +29,16 @@ class DataManager {
     private var animationStrings: Set<String>
     private var audioStrings: Set<String>
     private var audioGroups: Set<String>
+    //Stores the name of the last session accessed by the user
     private var lastSessionName: String?
 
-    //TODO: REALM INIT CAN FAIL
     init() {
-        // Not sure if this line should always be there
+        //Not sure if this line should always be there
```
Comment (anand-sundaram) [inline MIDIchlorians/MIDIchlorians/DataManager.swift]: Remove this comment too?

File: MIDIchlorians/MIDIchlorians/DataManager.swift [hunk 3]
```diff
@@ -273,23 +326,27 @@ class DataManager {
         }
     }
 
-    func getSamplesForGroup(group: String) -> [String] {
+    //Returns a list all audio files with a given audio group
+    func getAudiosForGroup(group: String) -> [String] {
         guard let audiosResultObject = realm?.objects(Audio.self).filter("audioGroup = %@", group) else {
```
Comment (anand-sundaram) [inline MIDIchlorians/MIDIchlorians/DataManager.swift:332]: Magic string?

**Review by anand-sundaram — COMMENTED** [review]
Nice comments for the specifications! Just remove the unnecessary/ambiguous ones

File: MIDIchlorians/MIDIchlorians/DataManager.swift (hunk 3 above)
Comment (advaypal) [reply MIDIchlorians/MIDIchlorians/DataManager.swift:332]: Can't really get rid of this

Comment (anand-sundaram) [reply MIDIchlorians/MIDIchlorians/DataManager.swift:332]: Ohh okay.

**Review by anand-sundaram — APPROVED** [review]
LGTM!

**Review by ngzhian — APPROVED** [review]

**Pull request merged**